cpu/cpupower_monitor.py

```diff
@@ -173,7 +187,8 @@ def test_idle_info(self):
             process.run('ppc64_cpu --smt=%s' % i, shell=True)
             process.run('ppc64_cpu --smt', shell=True)
             output = process.system_output(
-                'cpupower -c %s idle-info | grep offline' % i, shell=True).split()
+                'cpupower -c %s idle-info | grep offline' % i,
+                shell=True).split()
```

Comment:
i feel not relevant with change commit , can you please remove this and other places where it has style changes